libs/vkd3d/command.c

```diff
@@ -8675,7 +8687,7 @@ static void STDMETHODCALLTYPE d3d12_command_list_ClearUnorderedAccessViewUint(d3
                 /* If the clear color is 0, we can safely use the existing view to perform the
                  * clear since the bit pattern will not change. Otherwise, fill a scratch buffer
                  * with the packed clear value and perform a buffer to image copy. */
-                if (color.uint32[0])
+                if (color.uint32[0] || color.uint32[1] || color.uint32[2] || color.uint32[3])
```

Comment:
Do we clear unused components to 0 somewhere to avoid hitting fallback due to garbage values?

Reply:
mask_uint_clear_color should probably clear out unused components?

Reply:
Done.